package.json

```diff
@@ -37,93 +37,93 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "base64-js": "^1.2.0",
-    "brace": "0.7.0",
-    "deep-extend": "0.4.1",
+    "brace": "0.10.0",
```

Comment:
@shockey this actually needs to go. We have no need for brace here. In fact, I forgot about it, but brace + react-ace doubles the editor bundle size, since the versions differ, which is a big chunk of unnecessary code.